checkstyle-sonar-plugin/src/main/java/org/sonar/plugins/checkstyle/CheckstylePlugin.java

```diff
@@ -77,6 +84,15 @@ public List getExtensions() {
                                 .description(TREEWALKER_FILTERS_DESCRIPTION)
                                 .type(PropertyType.TEXT)
                                 .onQualifiers(Qualifiers.PROJECT, Qualifiers.MODULE).build(),
+                        PropertyDefinition.builder(CheckstyleConstants.CHECKER_TAB_WIDTH)
+                                .defaultValue(CheckstyleConstants.CHECKER_TAB_WIDTH_DEFAULT_VALUE)
```

Comment:
Should we not specify a default and allow it to be a blank string?
This way we use main repo's default and not overwrite our own. Description should be updated to say what a blank string means if we do this.

Reply:
Makes sense, I have left it empty by default and added a note in the description.